agent/api/container/container.go

```diff
@@ -66,8 +68,24 @@ const (
 
 	// TargetLogDriver is to show secret target being "LOG_DRIVER", the default will be "CONTAINER"
 	SecretTargetLogDriver = "LOG_DRIVER"
+
+	// Default RestartMaxAttempts OnFailure
+	DefaultRestartMaxAttemptsOnFailure = math.MaxUint32
 )
 
+const (
+	// not restarting at all
+	NEVER RestartPolicy = iota
```

Comment:
Any specific reason NEVER is all caps?

Reply:
Will keep this consistent with the other two, also as suggested, `Always` will be renamed as `UnlessTaskStopped`.